docs/standard/serialization/system-text-json-how-to.md

```diff
@@ -698,6 +709,177 @@ With this option, the `Summary` property of the `WeatherForecastWithDefault` obj
+* An exception is thrown if there are properties other than `$ref` in a JSON reference object.
+* The <xref:System.Text.Json.Serialization.ReferenceResolver> class defines the behavior of preserving references on serialization and deserialization. Create a derived class to specify custom behavior. For an example, see [GuidReferenceResolver](https://github.com/dotnet/dotnet-docs/blob/master/docs/standard/serialization/snippets/system-text-json-how-to-5-0/csharp/GuidReferenceResolver.cs).
+
+::: zone-end
+
+::: zone pivot="dotnet-core-3-1"
+<xref:System.Text.Json> 3.1 only supports serialization by value and throws an exception for circular references.
+::: zone-end
+
+## Quoted numbers
+
+::: zone pivot="dotnet-5-0"
+
+Some serializers encode numbers as JSON strings (surrounded by quotes). For example: `{"DegreesCelsius":"23"}` instead of `{"DegreesCelsius":23}`. To make <xref:System.Text.Json> write numbers in quotes or accept numbers in quotes, set <xref:System.Text.Json.JsonSerializerOptions.NumberHandling%2A?displayProperty=nameWithType> as shown in the following example:
+
+:::code language="csharp" source="snippets/system-text-json-how-to-5-0/csharp/QuotedNumbers.cs":::
+
+The  preceding code produces output similar to the following example:
+
+```output
+Output JSON:
+{
+  "date": "2020-10-19T14:40:27.1342475-07:00",
+  "temperatureC": "40",
+  "summary": "Hot"
+}
+Date: 10/19/2020 2:40:27 PM
+TemperatureC: 40
+Summary: Hot
+```
+
+When you use `System.Text.Json` indirectly through ASP.NET Core, quoted numbers are allowed because ASP.NET Core specifies [web default options](xref:System.Text.Json.JsonSerializerDefaults.Web).
+
+::: zone-end
+
+::: zone pivot="dotnet-core-3-1"
+<xref:System.Text.Json> 3.1 doesn' support serializing or deserializing numbers in quotes.
+::: zone-end
+
+## Copy a JsonSerializerOptions instance
+
+::: zone pivot="dotnet-5-0"
+There is a <xref:System.Text.Json.JsonSerializerOptions> constructor that lets you create a new instance with the same options as an existing instance, as shown in the following example:
+
+:::code language="csharp" source="snippets/system-text-json-how-to-5-0/csharp/CopyOptions.cs":::
+
+The preceding code produces output similar to the following example:
+
+```output
+Output JSON:
+{
+  "Date": "2020-10-20T18:38:33.4153344-07:00",
+  "TemperatureC": 40,
+  "Summary": "Hot"
+}
+```
+
+::: zone-end
+
+::: zone pivot="dotnet-core-3-1"
+A constructor that takes an existing instance is not available in `System.Text.Json` 3.1.
+::: zone-end
+
+## Web defaults for JsonSerializerOptions
+
+::: zone pivot="dotnet-5-0"
+There is a <xref:System.Text.Json.JsonSerializerOptions> constructor that creates a new instance with [the default options that are used for web apps](xref:System.Text.Json.JsonSerializerDefaults.Web), as shown in the following example:
```

Comment:
Technically the constructor accepts a JsonSerializerDefaults object. It could be Web or something else, depending on what the user specifies.

Reply:
Good point, I changed the wording so it doesn't imply this is the only possible usage of this constructor, though as a practical matter web defaults is the only scenario it's useful for at present.